lib/presentation/widget/upload_request_group/add_recipient_upload_request_group/add_recipients_upload_request_group_widget.dart

```diff
@@ -0,0 +1,231 @@
+                      fontWeight: FontWeight.normal,
+                      fontStyle: FontStyle.normal,
+                    ),
+                  )),
+            );
+          }),
+      floatingActionButtonLocation: FloatingActionButtonLocation.centerFloat,
+    );
+  }
+
+  Widget _buildRecipientsWidget(BuildContext buildContext) {
+    return Container(
+      margin: EdgeInsets.symmetric(vertical: 20.0, horizontal: 24.0),
+      child: Column(
+        children: [
+          Align(
+            alignment: Alignment.topLeft,
+            child: Text(
+              AppLocalizations.of(context).add_recipients,
+              style: TextStyle(fontSize: 16.0, color: AppColor.uploadFileFileNameTextColor),
+            ),
+          ),
+          SizedBox(
+            height: 17.0,
+          ),
+          TypeAheadFormField(
+              textFieldConfiguration: TextFieldConfiguration(
+                  controller: _typeAheadController,
+                  textInputAction: TextInputAction.done,
+                  decoration: InputDecoration(
+                      contentPadding: EdgeInsets.symmetric(vertical: 15),
+                      hintText: AppLocalizations.of(context).add_people,
+                      hintStyle:
+                          TextStyle(color: AppColor.uploadFileFileSizeTextColor, fontSize: 16.0),
+                      prefixIcon: Icon(
+                        Icons.person_add,
+                        size: 24.0,
+                      ))),
+              debounceDuration: Duration(milliseconds: 300),
+              suggestionsCallback: (pattern) async {
+                if (pattern.length >= 3) {
+                  return await _model.getAutoCompleteSharing(pattern);
+                }
+                return <AutoCompleteResult>[];
+              },
+              itemBuilder: (context, AutoCompleteResult autoCompleteResult) {
+                return ListTile(
+                  leading: LabelAvatarBuilder(autoCompleteResult
+                          .getSuggestionDisplayName()
+                          .characters
+                          .first
+                          .toUpperCase())
+                      .key(Key('label_avatar'))
+                      .build(),
+                  title: Text(autoCompleteResult.getSuggestionDisplayName(),
+                      style: TextStyle(fontSize: 14.0, color: AppColor.userTagTextColor)),
+                  subtitle: Text(autoCompleteResult.getSuggestionMail(),
+                      style: TextStyle(
+                          fontSize: 14.0, color: AppColor.userTagRemoveButtonBackgroundColor)),
+                );
+              },
+              onSuggestionSelected: (autoCompleteResult) {
+                _typeAheadController.text = '';
+                _model.addRecipientToList(autoCompleteResult as UserAutoCompleteResult);
+              },
+              noItemsFoundBuilder: (context) => _typeAheadController.text.isEmpty
```

Comment:
IMO, we should remove this `noItemsFoundBuilder ` because the `Unknow user` popup is redundant (it displays even though user just taps on the text field and still has not entered anything). Instead of that, we need more properties for `TypeAheadFormField`:

```
hideOnEmpty: true,
hideOnLoading: true
```

You can check the demo behavior from my PR: https://github.com/linagora/linshare-mobile-flutter-app/pull/547

Reply:
Yh I agree Unknown User is not really good to see I've made like you